tests/e2e_test.py

```diff
@@ -207,3 +208,17 @@ async def test_dune_to_local_job_run(self, mock_env, mock_dune_client):
             await conf.jobs[0].run()
 
         self.assertIn("No Query results found! Skipping write", logs.output[0])
+
+    @patch("src.config.load_dotenv")
+    @patch.dict(os.environ, {"DUNE_API_KEY": "test_key", "DB_URL": DB_URL})
+    @skipIf(not os.getenv("CI"), "this test only runs in CI")
```

Comment:
Neat! Learned something!

Reply:
So did I - what env vars are set in Github Actions!
Reference: https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/store-information-in-variables#default-environment-variables